NSS-Data-Analytics-Cohort-2/NSS-Data-Analytics-Cohort-2.github.io
# Style resume

A few changes here

1. Added in logic to the notebook for creating cohort.json to account for resumes. This has appropriate logic to handle file path names (lowercase)
2. Added in a disable class that greys out an icon / button
3. Added in JS logic to implement the resume for the modal and main card.. This design groups the "More" option and the resume into a single button to make more room on the card
4. Added in a fancy ribbon to specify if a student is trying to get hired!
5. Added in some JS sort and shuffle logic to make sure the students who are trying to get hired appear first in the website

To test: 
```bash
python3 -m http.server
```

Make sure that these students have ribbons (hire me) and show up first:  
> Landry, matt Barnes, Steven Decker, D. Mellow, Christina, Brant Ivey, Tracy Olson, Will McWilliams, Alan Brunlinger, Justinianlawcode, Brandes, Adam Tsuchiyama

Make sure that the carousel doesn't move the page as you scroll

Make sure that the buttons (resume and more) are combined into a single row

Make sure that some social icons are "disabled" - as well as resumes!


---

Thanks!
